plugins/juju/juju.plugin.zsh

```diff
@@ -123,5 +165,6 @@ jreld() {
 wjst() {
   local interval="${1:-5}"
   shift $(( $# > 0 ))
-  watch -n "$interval" --color juju status --relations --storage --color "$@"
+  watch -n "$interval" --color juju status --relations --color "$@"
```

Comment:
Is this one required? It seems like a nitpick that isn't worth a breaking change commit.